packages/nameguard-react/src/components/Share/Share.tsx

```diff
@@ -122,12 +124,13 @@ export function Share({ name }: ShareProps) {
                       Share NameGuard report
                     </Dialog.Title>
                     <div className="flex items-center right-0 inset-y-0 absolute pr-6 z-20">
-                      <button
+                      <Button
```

Comment:
For all the places that we have one of these "X" close icons, what do you think about using `IconButton` there instead of `Button`? This seems more semantically expressive for me.